docs/developer/http-api.md

```diff
@@ -35,6 +35,23 @@ The chain hash for the League of Entropy `quicknet` network running at a 3s freq
 
 Note that [the `fastnet` network has been deprecated.](/blog/2023/07/03/fastnet-sunset-quicknet-new/).
 
+### Testnet endpoints
+
+Furthermore, we also have "testnet endpoints" which are used as a test environment to test our latest changes, potentially helping you detect breaking changes or issues with our latest release before they hit mainnet.
+We recommend running your CI/CD against our testnet endpoints as well as our mainnet ones, in order to detect issues early.
```

Comment:
```suggestion
We recommend running your CI/CD against our testnet endpoints as well as our mainnet ones, in order to detect issues early.
In principle, there may be breaking changes between testnet and mainnet for a few weeks while we roll out new versions.
```

Reply:
hmm maybe this isn't _best_ here as you mention breaking changes above, though we ought to mention the fact that testnet and mainnet could be out of sync for a few weeks at a time

Reply:
I don't think that's the case for our relays, is it? We didn't have any breaking changes so far for relays